include/bluetooth/mesh/light_xyl_srv.rst

```diff
@@ -0,0 +1,118 @@
+
+    Your application is expected to hold the state memory and provide access to the state through the :c:struct:`bt_mesh_light_xyl_srv_handlers` handler structure.
+
+y: ``uint16_t``
+    The y state represents the y coordinate on the CIE1931 color space chart of a color light emitted by an element.
+    This is a 16-bit unsigned integer representation of a scale from 0 to 1 using the formula:
+
+    .. code-block:: console
+
+       CIE1931_y = (Light xyL y) / 65535
+
+    The y state power-up behavior is determined by the On Power Up state of the extended :ref:`bt_mesh_ponoff_srv_readme`:
+
+    * :c:enumerator:`BT_MESH_ON_POWER_UP_OFF` - The y state is set to Default y on power-up.
+    * :c:enumerator:`BT_MESH_ON_POWER_UP_ON` - The y state is set to Default y on power-up.
+    * :c:enumerator:`BT_MESH_ON_POWER_UP_RESTORE` - The y state is set to the last known y level.
+
+    Your application is expected to hold the state memory and provide access to the state through the :c:struct:`bt_mesh_light_xyl_srv_handlers` handler structure.
+
+Default xyL: :c:struct:`bt_mesh_light_xyl`
+    The Default xyL state is a meta state that controls the default Lightness, x and y level.
+    It is used when the light is turned on, but its exact state levels are not specified.
+
+    The memory for the Default xyL state is held by the model, and the application may receive updates on state changes through the
+    :c:member:`bt_mesh_light_xyl_srv_handlers.default_update` callback.
+
+    The Default Lightness state uses the configured lightness representation.
+
+Range: :c:struct:`bt_mesh_light_xyl_range`
+    The Range state is a meta state that determines the accepted x and y level range.
+    If the x or y level is set to a value outside the current Range, it is moved to fit inside the range.
+    If the Range changes to exclude the current x or y level, the level should be changed accordingly.
+
+    The memory for the Range state is held by the model, and the application may receive updates on state changes through the :c:member:`bt_mesh_light_xyl_srv_handlers.range_update` callback.
+
+Extended models
+================
+
+The Light xyL Server extends the following model:
+
+* :ref:`bt_mesh_lightness_srv_readme`
+
+The state of the extended Lightness Server model is for the most part bound to states in the Light xyL Server.
+The only exception is the Lightness range state, which is exposed to the application through the :c:member:`bt_mesh_light_xyl_srv_handlers.lightness_range_update` callback of the Light xyL Server model.
+
+Persistent storage
+===================
+
+The Light xyL Server stores the following information:
+
+* Any changes to the Default xyL and Range states
+* The last known Lightness, x, and y levels
+
+In addition the model takes over the persistent storage responsibility of the :ref:`bt_mesh_lightness_srv_readme` model.
+
+This information is used to reestablish the correct Light configuration when the device powers up.
```

Comment:
Could we write "light" configuration? Unless there is a specific reason why it was capitalized :)